src/i18ndude/catalog.py

```diff
@@ -535,8 +535,9 @@ def _write_messages(self, sort, msgstrToComment, sync):
             self._print_entry(
                 f, id, entry, msgstrToComment=msgstrToComment, sync=sync)
 
-        # File should end with a blank line
-        self._printToFile(f, False)
+        # File should end with a blank line, but we always print a full line
+        # including newline character anyway, so nothing special is needed.
+        # self._printToFile(f, False)
```

Comment:
True. I tend to leave code like this commented out with a warning, just in case later someone comes along and is tempted to put it back. But we can do without just. I have removed it now and force pushed it.